library/include/modules/References.h

```diff
@@ -0,0 +1,34 @@
+/*
```

Comment:
I don't think this license is necessary for new files. We have a license for the project as a whole